service/controller/app/resource/authtoken/create.go

```diff
@@ -0,0 +1,104 @@
+package authtoken
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/giantswarm/apiextensions/v3/pkg/label"
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/operatorkit/v4/pkg/controller/context/resourcecanceledcontext"
+	corev1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/giantswarm/app-operator/v2/pkg/annotation"
+	"github.com/giantswarm/app-operator/v2/pkg/project"
+	"github.com/giantswarm/app-operator/v2/service/controller/app/controllercontext"
+	"github.com/giantswarm/app-operator/v2/service/controller/key"
+)
+
+func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
+	cr, err := key.ToApp(obj)
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	if key.InCluster(cr) {
+		r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("app %#q in %#q uses InCluster kubeconfig no need to install webhook auth token", cr.Name, cr.Namespace))
+		r.logger.LogCtx(ctx, "level", "debug", "message", "canceling resource")
+		return nil
+	}
+
+	if key.AppName(cr) != key.ChartOperatorAppName {
```

Comment:
Only for chart-operator resource.

Reply:
Good we need this. Not for now but I think maybe we should drop the logs for non chart-operator CRs in the resources.

They add a lot of noise but don't add much value IMO.

Reply:
Agreed, I can jump into that tomorrow morning since we are in still the cool-down phase.

Reply:
Thanks that's a good cool-down task.